src/client.jl

```diff
@@ -292,7 +292,12 @@ function set_header_content_type(ctx::Ctx, ctypes::Vector{String})
 end
 
 set_param(params::Dict{String,String}, name::String, value::Nothing; collection_format=",") = nothing
-function set_param(params::Dict{String,String}, name::String, value; collection_format=",")
+function set_param(params::Dict{String,String}, name::String, value; collection_format=",", style="form", is_explode=false)
+    deep_explode = style == "deepObject" && is_explode
```

Comment:
Not, sure. This is just to be extra sure just in case (unlikely) the spec does define explode=false

Reply:
I've added a suggestion externalizing the default `explode` value to a method. Also added an additional `location` kwarg to `set_param`. It will allow us to implement a more complete implementation gradually.